downloads/bn-BD/github-git-cheat-sheet.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: cheat-sheet
+title: গিটহাব গিট চিট শীট
+byline: গিট হচ্ছে একটি ওপেন সোর্স ড্রিস্ট্রিবিউটেড ভার্শন কন্ট্রোল সিস্টেম যা আপনার ল্যাপটপ বা ডেস্কটপে গিটহাব সুবিধাদি প্রদান করে। এই চিত শীটে সাধারনভাবে ব্যবহৃত গিট কমান্ড লাইনের নির্দেশনা দেওয়া আছে । 
+leadingpath: ../../
```

Comment:
@bolaram currently this line is what's causing the css not to load. if you change this to `../../../` then it should work!